showcase/radar-chart/animated-radar-chart.js

```diff
@@ -72,6 +72,9 @@ export default class AnimatedRadar extends Component {
               text: {
                 opacity: 0
               }
+            },
+            labels: {
```

Comment:
Ideally yes, but when user passes a custom `style` prop, [`style.labels` here](https://github.com/zachguo/react-vis/blob/03cfbe36b7e7c9fa018ad9b2752c712ba6e50bcd/src/radar-chart/index.js#L190) would be undefined.